cli/generators/crud/templates/components/addPage.js

```diff
@@ -0,0 +1,72 @@
+import React, { PropTypes } from 'react';
+import { browserHistory } from 'react-router';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import toastr from 'toastr';
+import * as <%= name %>Actions from '../../actions/<%= name %>Actions';
+import <%= ucName %>Form from './<%= ucName %>Form';
+
+class <%= ucName %>AddPage extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      <%= name %>: Object.assign({}, props.<%= name %>),
+      saving: false
+    };
+
+    this.handleSave = this.handleSave.bind(this);
+  }
+
+  handleSave(<%= name %>) {
+    this.setState({ saving: true });
+    this.props.actions.create<%= ucName %>(<%= name %>)
+      .then(() => {
+        this.setState({ saving: false });
+        toastr.success('<%= ucName %> created successfully');
+        browserHistory.push('/app/<%= pluralizedName %>');
+      })
+      .catch(error => {
+        this.setState({ saving: false });
```

Comment:
Related to this, I'm doing some tests, research on a client project, because i'm having some troubles with the then(), catch() and setting states, for example. So I'll be providing some input next week I think. Thanks @AlphaGit and @matiasbeckerle